# Internal Temperature [Â°C]
float32 temp

# delta velocity output [m/s]
geometry_msgs/Vector3 delta_vel

# delta angle output [degrees]
geometry_msgs/Vector3 delta_angle

int32 count
